types/mapbox__mapbox-sdk/mapbox__mapbox-sdk-tests.ts

```diff
@@ -82,6 +82,21 @@ staticMapService.getStaticImage({
     height: 16,
     position: 'auto',
     overlays: [
+        {
+            path: {
+                coordinates: [
+                    [8.1298828125, 10.098670120603392],
+                    [9.4921875, 15.792253570362446],
+                    [11.77734375, 14.179186142354181],
+                    [11.513671874999998, 11.6522364041154]
+                ],
+                strokeColor: 'ff0000',
+                strokeWidth: 10,
+                strokeOpacity: 0.4,
+                fillColor: '000',
+                fillOpacity: 0.75
+        }
```

Comment:
This is so minor so apologies for calling it out, but just add in another tab for whitespace here since it closes `path`

Reply:
Great catch! Happy to add it in. :)